piqe_ocp_lib/api/resources/ocp_operators.py

```diff
@@ -306,6 +306,24 @@ def __init__(self, kube_config_file):
         self.kind = "CatalogSource"
         self.catalog_source_obj = self.dyn_client.resources.get(api_version=self.api_version, kind=self.kind)
 
+    def create_catalog_source(self, cs_name, image, namespace="openshift-marketplace"):
+        spec_dict =  {"displayName": "Optional operators",  "icon": {"base64data": "", "mediatype": ""} , 
```

Comment:
If spec_dict is not going to be prameterized, then there is no benefit in separating the two. It looks like for this method the only two required arguments are cs_name and image, in that case just put the whole dict together under one object. Maybe make displayName and publisher as optional arguments and give them the currently assigned values by default.

Reply:
Done, please check.